porter/schema.py

```diff
@@ -125,6 +126,18 @@ class GetUrsulas(BaseSchema):
         ),
     )
 
+    min_version = VersionString(
```

Comment:
We need to update the README to add this optional parameter.

Filed #75 which can be addressed in a separate PR.

Reply:
Added small note in README for get_ursulas